### ISSUE TITLE
Have clearer way of managing offline lyrics

### ISSUE THREAD

**Issue by Miepee** [f-leoni/lyrics2#16]
Are currently only favourited lyrics saved offline? I'm a little confused by it, because I'd like some songs to be offline but not favourited, but I can't seem to find a way to then still access them.

**Comment by f-leoni** [comment]
Yep, adding to Favorites is the current way of "saving offline" the lyrics. 
I'll try and think for a feature for saving but not adding to favorites

**Comment by Miepee** [comment]
Could have a seperate "offline" entry in the navbar or something, that has the downloaded songs sorted by artist name and then song name.
Because using unsorted favorites for that purpose gets messy real fast.
